hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -40,6 +40,12 @@ private ReconConfigKeys() {
       "ozone.recon.http-address";
   public static final String OZONE_RECON_HTTPS_ADDRESS_KEY =
       "ozone.recon.https-address";
+  public static final String OZONE_RECON_ADDRESS_DEFAULT =
+      "recon:9891";
+  public static final String OZONE_RECON_HTTP_ADDRESS_DEFAULT =
+      "0.0.0.0:9888";
+  public static final String OZONE_RECON_HTTPS_ADDRESS_DEFAULT =
+      "0.0.0.0:9889";
```

Comment:
Reorder the variables. Make them in the order of KEY / DEFAULT / KEY / DEFAULT / KEY / DEFAULT.